core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/backoff/FullJitterBackoffStrategy.java

```diff
@@ -62,7 +62,7 @@ private FullJitterBackoffStrategy(BuilderImpl builder) {
     public Duration computeDelayBeforeNextRetry(RetryPolicyContext context) {
         int ceil = calculateExponentialDelay(context.retriesAttempted(), baseDelay, maxBackoffTime);
         // Minimum of 1 ms (consistent with BackoffStrategy.none()'s behavior)
-        return Duration.ofMillis(random.nextInt(ceil) + 1L);
+        return ceil == 0 ? Duration.ofMillis(1L) : Duration.ofMillis(random.nextInt(ceil) + 1L);
```

Comment:
Weird we use 1ms in `BackoffStrategy.none()`. Maybe we should just use zero here, since if someone sets the max of zero, they'd probably be surprised to see 1 ms?

Reply:
Done.